### ISSUE TITLE
Error: Cannot find module 'tls'

### ISSUE THREAD

**Issue by yuripourre** [playht/playht-nodejs-sdk#35]
Hello, I am following the steps in the documentation.

I am trying to add PlayHT to a project using next.js and webpack 5 but I am getting this error when I tried to call the listVoices method.

I get this error:

```
Error: Cannot find module 'tls'
...
 ⚠ ./node_modules/playht/dist/esm/index.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
```

I tried this config but it didn't work:
```js
const nextConfig = {
    webpack: (config) => {
        config.resolve.fallback = { tls: false };
        return config;
    },
};
export default nextConfig;
```

Do you guys have any ideas what I am doing wrong?

**Comment by ttanida** [comment]
I have the same problem.
The playHT node sdk doesn't seem to work with next.js.

Other people have the same problem, see: https://github.com/playht/playht-nodejs-sdk/issues/32#issuecomment-2095592559

I have tried a lot of different things, but nothing has worked so far.

**Comment by yuripourre** [comment]
@ttanida I end up implementing all the calls using axios. I could successfully get the list of voices (V1) but apparently the text to speech endpoint doesn't work with a free account anymore.

As per: https://docs.play.ht/discuss/65e4b810845339000fd46959

It's frustrating because I would like to test it before paying for it.

**Comment by ttanida** [comment]
@yuripourre, you could try the Hacker plan, I think it is a reasonable price for trying out the API (https://play.ht/pricing/).

The problem with using axios for me is that, if I see correctly, I can't do input text streaming that way (only output audio streaming), since I have to send the whole text in the call. So I could just use the openAI TTS API instead, which also supports audio streaming (and I found the openAI voices to be better than playht).

Also, there is a 25 RPM rate limit on the API endpoint, which would be too low for my use case.

**Comment by srelbo** [comment]
@AbdelhamidGamal Can you please take a look at this one? Thanks!

**Comment by yuripourre** [comment]
> The problem with using axios for me is that, if I see correctly, I can't do input text streaming that way (only output audio streaming).

That's right, it doesn't support stream. But for my use case is good enough.

**Comment by OrkhanGG** [comment]
Solved:
https://stackoverflow.com/questions/77830195/playht-integration-with-nextjs/78667405#78667405
